UQ-eLIPSE/crucible
# Bugfix/team24-39

Fixed the HTML rendering without using `v-html` directive and using the recommended DOMParsing.

Added E2E tests for creating and checking the content.

To test:

1. `cd client && yarn test:e2e`
2. Go to https://localhost:5173
3. Login
4. Create a test collection then a test collection with internal document resource.
5. Verify that you don't see any HTML tags displaying.
